components/x-podcast-launchers/src/map-concept-to-acast-series.js

```diff
@@ -0,0 +1,10 @@
+import { brand } from '@financial-times/n-concept-ids';
+
+export default function mapConceptToAcastSeries(conceptId) {
```

Comment:
I think a comment here explaining why one thing is weird (and pref. who is responsible for it being weird) would be useful

Reply:
@i-like-robots how do you mean? As in why does only one `conceptId` have a mapping?

Reply:
I looked at this in isolation but looking at where it is implemented makes more sense.